VoteCounter/src/GUI.cs

```diff
@@ -29,8 +29,48 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
             {
                 PostList postList = new PostList(url, startingPostNumber, endingPostNumber);
 
+                //make a List
+                List<Player> players = new List<Player>();
+
+                //set the players
+                foreach (DataGridViewRow row in listPlayers.Rows)
+                {
+                    List<string> nicks = new List<string>();
+                    string name = "";
+                    for (int i = 0; i < listPlayers.Columns.Count; i++)
```

Comment:
I think we already do skip over it because we explicitly refer to columns by name. I could throw in a blank if(column=0){//do nothing} if you'd like?